docs/ha-considerations.md

```diff
@@ -83,23 +83,32 @@ There are some placeholders in `bash` variable style to fill in:
 
 The above `keepalived` configuration uses a health check script `/etc/keepalived/check_apiserver.sh` responsible for making sure that on the node holding the virtual IP the API Server is available. This script could look like this:
 
-```
+```bash
 #!/bin/sh
 
 errorExit() {
     echo "*** $*" 1>&2
     exit 1
 }
 
-curl --silent --max-time 2 --insecure https://localhost:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://localhost:${APISERVER_DEST_PORT}/"
-if ip addr | grep -q ${APISERVER_VIP}; then
-    curl --silent --max-time 2 --insecure https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/"
+# replace this with the virtual IP address negotiated between the `keepalived` cluster hosts
+APISERVER_VIP=${APISERVER_VIP_VALUE}
+# replace this with the port through which Kubernetes will talk to the API Server
+APISERVER_DEST_PORT=${APISERVER_DEST_PORT_VALUE}
+
```

Comment:
there is no point having both VALUE and the non- VALUE variables.

Reply:
You mean delete `APISERVER_VIP`? It's needed in the script below. If you want to put `APISERVER_VIP` in the script below and let user replace that string, then user need to replace it multiple times. So why not declare it here so that user can just replace the value of if once?

Reply:
remove *_VALUE vars.

Reply:
sorry, I didn't get it, why we need this placeholders? shouldn't the env var "APISERVER_VIP" and "APISERVER_DEST_PORT" are enough, they can be replaced by the actual IP addr or port, right?